vms/platformvm/state/state.go

```diff
@@ -214,7 +214,7 @@ type stateBlk struct {
  *   '-- lastAcceptedKey -> lastAccepted
  */
```

Comment:
Can we update this comment to stay in-line with the new `validator` and `subnetValidator` formats?

Reply:
[9928104](https://github.com/ava-labs/avalanchego/pull/1262/commits/9928104476267aa6348fb36bfa5f83b9adb9d9e2)